Linux - Workouts 1 - 3 Improvements (#2096)
enkidevs/curriculum: linux/user-and-file-management/bash-environment/common-environment-variables.md

---
author: jfarmer

aspects:
  - introduction
  - workout

type: normal

category: must-know

tags:
  - introduction
  - linux
  - environment
  - variables

---

# Common Environment Variables

---
## Content

On Linux (and other Unix-based operating systems), common environment variables include:

> - `HOME`, which contains the current user's home directory

Reviewer: ```suggestion
- `HOME`, which contains the current user's home directory. That is the top-level directory to most other directories
```

> - `PATH`, which contains a list of directory paths. When a user types a command without providing the full path, `bash` will look at the directories in the `PATH` environment variable to see if they contain the given command.

Reviewer: ```suggestion
- `PATH`, which contains a list of directories where to look for a command. When a user types `echo` in the terminal, Linux finds the `echo` program in one of the directories listed in `PATH` and executes it.
```

> - `PS1`, which specifies how the prompt is displayed

Reviewer: `PWD` is probably way more common than `PS1`

```suggestion
- `PWD `, which holds the path to the directory you're currently in. This is usually called the "working directory".
```
